cmd/oj/main.go

```diff
@@ -290,7 +296,12 @@ func run() (err error) {
 		var f *os.File
 		for _, file := range files {
 			if f, err = os.Open(file); err == nil {
-				_, err = p.ParseReader(f, write)
+				// TBD
```

Comment:
Is this a TODO? Something forgotten?

Reply:
I'm surprised. This is the first time someone has actually participated. Thank you. TBD comment removed.

Reply:
You are welcome.

I started watching the repository a few weeks ago